File: MaxAndMin.py
def high_and_low(numbers):
    nums = numbers.split()
    maxInt = int(nums[0])
    minInt = int(nums[0])
    for num in nums[1:]:
        maxInt = max(maxInt, int(num))
        minInt = min(minInt, int(num))
    return f"{maxInt} {minInt}"

File: JS/MaxAndMin.js
function highAndLow(numbers) {
    var nums = numbers.split(" ")
    var max = parseInt(nums[0]);
    var min = parseInt(nums[0]);
    for (var i = 1; i < nums.length; i++) {
        max = Math.max(max, parseInt(nums[i]))
        min = Math.min(min, parseInt(nums[i]))
    }
    return max + " " + min
}

highAndLow("4 5 29 54 4 0 -214 542 -64 1 -3 6 -6")